src/components/DpDataTableExtended/DpDataTableExtended.vue

```diff
@@ -65,24 +65,12 @@
         </slot>
       </template>
       <template
-        v-for="el in filteredFields"
-        v-slot:[el.field]="element">
+        v-for="el in [...filteredFields, { field: 'expandedContent' }, { fields: 'flyout' }]"
```

Comment:
If i correctly get the idea, i think it should be: 
 
```suggestion
        v-for="el in [...filteredFields, { field: 'expandedContent' }, { field: 'flyout' }]"
```